### PR TITLE
Add issue templates

### LINKED ISSUE #26: Add issue templates
Add [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) to this repository.

### PR DESCRIPTION
**Pull request by artis3n**
- add issue templates
- make lint

Adding issue templates following the examples in <https://github.com/ansible/ansible/tree/devel/.github/ISSUE_TEMPLATE>.

Closes #26

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+      label: Issue Type
+      description: >
+        Please select the single available option in the drop-down.
+
+        <details>
+          <summary>
+            <em>Why?</em>
+          </summary>
+
+          We would do it by ourselves but unfortunatelly, the curent
```
Comment (mamercad) [inline .github/ISSUE_TEMPLATE/bug_report.yml:45]: Spelling: "unfortunately" and "current"

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        libyaml = True
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Configuration
+      description: >-
+        Paste verbatim output from `ansible-config dump --only-changed -t all` below, under the prompt line.
+        (if using a version older than ansible-core 2.12 you should omit the '-t all')
```
Comment (mamercad) [inline .github/ISSUE_TEMPLATE/bug_report.yml:98]: We're not [testing < 2.14](https://github.com/digitalocean/ansible-collection?tab=readme-ov-file#tested-with-versions) and it's EOL, should we mention this at all?

**Review by mamercad — CHANGES_REQUESTED** [review]
Nice; couple of typos, I'll leave the question up to you.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (artis3n) [reply .github/ISSUE_TEMPLATE/bug_report.yml:98]: Thanks, I changed some lines from what ansible/ansible uses, but did copy some large chunks. I agree, we don't need to mention in our project.

**Pull request merged**